src/rust/engine/fs/store/src/lib.rs

```diff
@@ -1599,7 +1599,8 @@ impl Store {
         let res = async move {
             let directory = store.load_directory(digest).await.map_err(|e| {
                 e.enrich(&format!(
-                    "Could not walk unknown directory at {path_so_far:?}"
+                    "Could not walk unknown directory at {}",
```

Comment:
The Debug output includes superfluous quotes. The `display()` output is a tiny bit nicer.